util/fipstools/acvp/acvptool/acvp.go

```diff
@@ -194,6 +194,16 @@ func looksLikeHeaderElement(element json.RawMessage) bool {
 	return len(headerFields.URL) > 0 || len(headerFields.Algorithm) == 0
 }
 
+func looksLikeTestHeaderElement(element json.RawMessage) bool {
+    var testheaderFields struct {
+        Version string `json:"acvVersion"`
+    }
+    if err := json.Unmarshal(element, &testheaderFields); err != nil{
+        return false
+    }
+	return len(testheaderFields.Version) > 0
+}
+
```

Comment:
Nit: `Version string `json:"acvVersion"`` can be consolidated in `looksLikeHeaderElement`.

Reply:
Just found that BoringSSL added a commit 5799ebfe5f895ecd29fcd699789854fc1790d6cd that takes care of the test version string after I started work on this. Deleted my redundant code 👍